<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidget</class>
 <widget class="QDockWidget" name="DockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kaavamääräysryhmät</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="new_btn">
        <property name="toolTip">
         <string>Luo uusi kaavamääräysryhmä</string>
        </property>
        <property name="text">
         <string>Luo</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="edit_btn">
        <property name="toolTip">
         <string>Muokkaa valittua kaavamääräysryhmää</string>
        </property>
        <property name="text">
         <string>Muokkaa</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delete_btn">
        <property name="toolTip">
         <string>Poista valittu kaavamääräysryhmä</string>
        </property>
        <property name="text">
         <string>Poista</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="remove_all_btn">
      <property name="text">
       <string>Poista kaikki ryhmät valituilta kohteilta</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="add_all_btn">
      <property name="text">
       <string>Lisää valitut ryhmät valituille kohteille</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QgsFilterLineEdit" name="search_box">
      <property name="showSearchIcon">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="regulation_group_list"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFilterLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qgsfilterlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
